rules-tests/Symfony73/Rector/New_/RegexToSlugConstraintRector/Fixture/replace_all_params.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Symfony\Tests\Symfony73\Rector\New_\RegexToSlugConstraintRector\Fixture;
+
+use Symfony\Component\Validator\Constraints\Regex;
+
+class ReplaceAllParams
+{
+    public function createConstraint()
+    {
+        return new Regex(pattern: '/[a-z0-9]+(?:-[a-z0-9]+)*/', message: 'some message', htmlPattern: null, match: null, normalizer: null, groups: ['Default'], payload: null, options: []);
```

Comment:
Maybe i should skip if `htmlPattern`, `match` or `normalizer` are part of the params?